docs/troubleshooting.rst

```diff
@@ -0,0 +1,23 @@
+Troubleshooting
+###############
+
+.. _troubleshooting:
+
+We are doing the best we can so you do not have to read this section.
+
+In case you do, please find the common problems you may have below.
```

Comment:
That said, we've included solutions (or at least explanations) for some common problems below.